docs/VnVPlan/VnVPlan.tex

```diff
@@ -121,11 +122,18 @@ \subsection{Verification and Validation Team}
 
 \subsection{SRS Verification Plan}
 
-\wss{List any approaches you intend to use for SRS verification.  This may include
-	ad hoc feedback from reviewers, like your classmates, or you may plan for
-	something more rigorous/systematic.}
+SRS verification will be done by the core developers of the project, as well as by the TAs of the
+course. To verify the contents of the
+\href{https://github.com/arkinmodi/project-sayyara/blob/main/docs/SRS/SRS.pdf}{SRS}, we will
+compare the requirements stated by the stakeholder with the requirements stated in the SRS.
+Furthermore, we will show our SRS to our stakeholder for their review. We believe that this is
```

Comment:
stakeholders? If you mean Nabeel, you should clarify since we are also stakeholders